python/ffsim/tenpy/util.py

```diff
@@ -0,0 +1,60 @@
+# (C) Copyright IBM 2024.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+from __future__ import annotations
+
+from tenpy.networks.mps import MPS
+from tenpy.networks.site import SpinHalfFermionSite
+
+import ffsim
+
+
+def product_state_as_mps(norb: int, nelec: int | tuple[int, int], idx: int) -> MPS:
```

Comment:
Let's refactor this function to have this API:
```python
def bitstring_to_mps(bitstring: tuple[int, int]) -> MPS
```
Here the input is a tuple of the alpha string and the beta string: `(str_a, str_b)`. I think this API is more straightforward.

Reply:
Sounds good, using this change I don't need `nelec` anymore. Also, if I take `bitstring: tuple[str, str]`, I don't need `norb` either, since I can use the `len()` function